kratos/modified_shape_functions/modified_shape_functions.h

```diff
@@ -296,11 +296,11 @@ class KRATOS_API(KRATOS_CORE) ModifiedShapeFunctions
     * @return rInterfaceShapeFunctionValues: Matrix containing the computed shape function values.
     * @return rInterfaceShapeFunctionsGradientsValues: std::vector containing the shape functions gradients values.
     * @return rInterfaceWeightsValues: Vector containing the Gauss pts. weights (already multiplied by the Jacobian).
-    * @param rInterfacesVector: std::vector of intersection point based geometries where the values are to be computed.
-    * @param rParentGeometriesVector: std::vector of subdivisions point based parent geometries.
-    * @param rInterfacesParentIdsVector: std::vector containing the parent ids of each interface geometry.
-    * @param rPmatrix: reference to the interface interpolation matrix
-    * @param IntegrationMethod: Desired integration quadrature.
+    * @param rInterfacesVector stdvector of intersection point based geometries where the values are to be computed.
```

Comment:
Why you are taking out the std::vector double points? Is there anyway to keep it?

Reply:
I used a script to speed up the work, I checked out, but maybe I made some mistake